docker-compose.yml

```diff
@@ -10,21 +11,23 @@ services:
             - "993:993"
             - "995:995"
         depends_on:
-            - redis
+            - cache
             - mongo
-
-    redis:
-        network_mode: host
+        volumes:
+            - ./data/dbs.toml:/wildduck/config/dbs.toml
+            - ./data/api.toml:/wildduck/config/api.toml
+    cache:
         image: redis:alpine
+        container_name: cache
         volumes:
             - /data
         expose:
             - 6379
     mongo:
-        network_mode: host
         image: mongo
+        container_name: mongo
         restart: always
         volumes:
             - /data/db
-        ports:
-            - 27017
+        expose:
```

Comment:
The mongo dockerfile already exposes port 27017. I agree with you that it shouldn't be published by default, so I think it can just be removed.

Reply:
Yes, it can, I had published to check locally, but then hid and just didn't remove to keep things explicit, will remove.